docs/user-guide/release-notes/master.rst

```diff
@@ -55,6 +55,27 @@ Known Issues
 Upgrade Instructions for 2.6.x --> master
 -----------------------------------------
 
+Upgrade the packages using:
```

Comment:
I wasn't very clear here. I mean replace `Upgrade the packages using:` with `Upgrade the packages using:::`

The `:::` additionally below will show as a single colon on a line when rendered.